src/m365/spo/commands/file/FileSharingLinkUtil.ts

```diff
@@ -0,0 +1,33 @@
+import request, { CliRequestOptions } from "../../../../request";
+import { formatting } from "../../../../utils/formatting";
+import { urlUtil } from "../../../../utils/urlUtil";
+import { GraphFileDetails } from "./GraphFileDetails";
+
+export class FileSharingLinkUtil {
+  public static readonly allowedScopes: string[] = ['anonymous', 'users', 'organization'];
```

Comment:
I'm not sure if we should centralize this one yet. Some commands don't support all scopes yet.